Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/ParsingUtilitiesScriptEventExtension.cpp

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ScriptCanvas/Core/Graph.h>
+#include <ScriptCanvas/Core/GraphSerialization.h>
+#include <ScriptCanvas/Grammar/ParsingUtilitiesScriptEventExtension.h>
+#include <ScriptCanvas/Libraries/Core/FunctionDefinitionNode.h>
+#include <ScriptCanvas/Variable/VariableBus.h>
+#include <ScriptCanvas/Variable/VariableData.h>
+#include <ScriptEvents/ScriptEventsMethod.h>
+
+namespace ParsingUtilitiesScriptEventExtensionCpp
+{
+    using namespace ScriptEvents;
+    using namespace ScriptCanvas;
+    using namespace ScriptCanvas::Nodes::Core;
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether